packages/utilities/psammead-locales/moment/ar.test.js

```diff
@@ -0,0 +1,259 @@
+/* eslint-disable */
+
+import moment from 'moment';
+import './ar';
+
+moment.locale('ar');
+
+// This asset overrides the gunit assertion done in the moment codebase.
+// Format and styling of this file has been keep consistent with the official moment tests.
+// An example of these tests can be seen at https://github.com/moment/moment/blob/develop/src/test/locale/en-gb.js
+const assert = { equal: (val1, val2) => expect(val1).toEqual(val2) };
+
+const months = [
+  'يناير/ كانون الثاني',
+  'فبراير/ شباط',
+  'مارس/ آذار',
+  'أبريل/ نيسان',
+  'مايو/ أيار',
+  'يونيو/ حزيران',
+  'يوليو/ تموز',
+  'أغسطس/ آب',
+  'سبتمبر/ أيلول',
+  'أكتوبر/ تشرين الأول',
+  'نوفمبر/ تشرين الثاني',
+  'ديسمبر/ كانون الأول',
```

Comment:
Have you confirmed that these render correctly (== like legacy PAL pages) inside a `dir=rtl` element?

Reply:
yes i have